azext_iot/central/command_map.py

```diff
@@ -219,3 +219,26 @@ def load_central_commands(self, _):
         cmd_group.command("resume", "resume_job")
         cmd_group.command("get-devices", "get_job_devices")
         cmd_group.command("rerun", "rerun_job")
+
+    with self.command_group(
+        "iot central device edge children", command_type=central_device_ops
+    ) as cmd_group:
+        cmd_group.command("list", "list_children")
```

Comment:
Is children list not preview?

Reply:
children can be listed without "relationships API" too by reading "scope" flag in device twin. So it's always available no matter what api-version being used. Instead, ADD and REMOVE are only available for api-version > 1.1-preview as we can't write on twins from outside Central and relationships API is the only approach (still in preview).